phylanx/include/util.hpp

```diff
@@ -7,6 +7,7 @@
 #define PHYLANX_UTIL_HPP
 
 #include <phylanx/config.hpp>
+#include <phylanx/util/instrumentation/performance_data.hpp>
```

Comment:
We usually try to match the namespace with the directory structure. I think we can safely put the new header file directly into `phylanx/util` for now to make those match.

Reply:
Fixed.